docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField
+
+Input properties, such as `InputColor` and `InputPointSize`, are used when you want to set new input texts differently from the existing text settings when the new text inputs.
+
+Especially, these properties should be used with some specific conditions.
+
+For instance, they're used when `TextChanged` signal is emitted or a button is clicked.
+
+The following example illustrates how to use Input properties:
+
+```csharp
+TextField field = new TextField();
+field.TextColor =
+field.TextChanged += (obj, e) => {
+    e.TextField.InputColor = Color.Yellow;
+};
+```
+
+<a name="textField5"></a>

```

Comment:
Is this a newly added anchor?

Reply:
Could you tell me more details?

I just added this tag because the original tag of ### TextField Properties is <a name="textField4"></a>. 
So, I want to add new chapter between ### Align Text in TextField and ### TextField Properties .

Reply:
\<a name="textField5"></a>

This anchor is used for the hyperlink from other place to here.
For example, in the original file (before this update),
if you insert a hyper link in somewhere of dotnet/guides/nui/text.md as \[sample text](#textField5),
then when you click "sample text" it will jump to the line 404. 


With this update,
to jump into "TextField Properties" section, the hyperlinks should change  \[sample text](#textField5) to  \[sample text](#textField6),

So usually we keep the original "name"s without update, and give new names to the new section.

Or,

as the md file generates the name for each section automatically using the headings,
we use the automatically generated names to jump into some section as like 
\[sample text](#textfield-properties).

I think @annie-samsung's doubt is : this update seems to change the existing anchor.
As it is recommended no to change the hyperlink anchor,
it would be better to keep \<a name="textField4"></a> before "### TextField Properties",
and change the anchor in front of "Use Input Properties in TextField".

In my view, you don't even need to add \<a name="textField#"></a> in front of \### Use Input Properties in TextField, 
as we can use "#use-input-properties-in-textfield" if you want to make a hyperlink which jumps into here.

Reply:
Oh I got it. Now I understand it.
The "name" should be kept and I need to add a new name.
I'll modify the name to "input-properties".
Thank you for your detailed explanation.